### ISSUE TITLE
SSD-VGG: Loss nan

### ISSUE THREAD

**Issue by hoangcuongbk80** [qfgaohao/pytorch-ssd#127]
Hi,

I trained the SSD for my own dataset using VGG and get loss nan as below:

![88891938-4d04f100-d244-11ea-803f-a43f13565a04](https://user-images.githubusercontent.com/10355081/89096109-3c838080-d3d4-11ea-826a-ed844fe6942b.png)

I trained my own data on Jetson Xavier. It's ok if training using MobileNet. Any suggestion to make it work with VGG?

**Comment by francescooliva** [comment]
I had your same error, I completed the training using a lower learning rate i.e --lr 0.0001

**Comment by hoangcuongbk80** [comment]
@francescooliva  Thank you for sharing your experience. I'll try it.

**Comment by francescooliva** [comment]
Also I can see from your screenshot it looks like you are using mobilenet training parameters, or am I wrong? check in the Readme the Vgg model training instructions:
`python train_ssd.py --datasets ~/data/VOC0712/VOC2007/ ~/data/VOC0712/VOC2012/ --validation_dataset ~/data/VOC0712/test/VOC2007/ --net vgg16-ssd --base_net models/vgg16_reducedfc.pth  --batch_size 24 --num_epochs 200 --scheduler "multi-step” —-milestones “120,160” `
